foundry.toml

```diff
@@ -100,7 +90,7 @@
   wrap_comments = true
 
 [rpc_endpoints]
-  arbitrum = "https://arbitrum-mainnet.infura.io/v3/${API_KEY_INFURA}"
+  arbitrum_one = "https://arbitrum-mainnet.infura.io/v3/${API_KEY_INFURA}"
```

Comment:
Shouldn't this be renamed to "arbitrum" as well?

Reply:
Yep, it should. I will fix later today (have some staging changes locally.)